stock_move_line_auto_fill/views/stock_picking_type_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="view_picking_type_form_assignment">
+        <field name="name">stock.picking.type.assignment</field>
+        <field name="model">stock.picking.type</field>
+        <field name="inherit_id" ref="stock.view_picking_type_form"/>
+        <field name="arch" type="xml">
+            <field name="use_existing_lots" position="after">
+                <field name="auto_fill_operation"/>
+                <field name="avoid_lot_assignment" groups="stock.group_production_lot"/>
+            </field>
```

Comment:
Here is an improvement
https://github.com/tegin/stock-logistics-workflow/pull/1/files

Reply:
Thank you!